app/models/user.rb

```diff
@@ -60,7 +68,19 @@ def password_reset_expired?
   end
 
   def feed
-    microposts
+    Micropost.where("user_id IN (?) OR user_id = ?", following_ids, id)
```

Comment:
`t.where("user_id IN (?) OR user_id = ?", following_ids, id)` -> viết scope nhé

cơ mà chỗ này đó, ko cần or mô, em gộp value của ` following_ids, id` thành 1 cái array là dc
` following_ids, id`